scripts/regridding/regrid_example.py

```diff
@@ -58,7 +58,7 @@ def regrid_example(case_name, input_climo_loc, output_loc,
 
         #loop over regridding targets:
         for target in target_list:
-
+            print(f"DEBUG: target = {target}")
```

Comment:
Can you comment this print statement out and add an extra comment stating that it needs to be added to a debug log?